compiler/lsp-tests/src/Main.hs

```diff
@@ -0,0 +1,72 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+module Main (main) where
+
+import DA.Bazel.Runfiles
+import qualified Data.Text as T
+import Language.Haskell.LSP.Test
+import Language.Haskell.LSP.Types
+import System.FilePath
+import System.IO.Extra
+import Test.Tasty
+import Test.Tasty.HUnit
+
+import Daml.Lsp.Test.Util
+
+main :: IO ()
+main = do
+    damlcPath <- locateRunfiles $
+        mainWorkspace </> "daml-foundations" </> "daml-tools" </>
```

Comment:
Why not just `daml-foundations/daml-tools/da-hs-damlc-app/...` - using `</>` doesn't really help but is more verbose.

Reply:
At this point I’m slightly traumatized from issues caused by mixing `/` and `\` on Windows so I try to avoid it where I can :slightly_smiling_face: